intro-sql-access.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Query your database using SQL'
+description: 'Run SQL queries over HTTP directly in PostgreSQL'
+author: Noémi Ványi
+date: 08-31-2023
+published: false
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-full-text-search.png
+  alt: SQL access to Xata
+slug: sql-query-over-http-intro
+---
+
+We are excited to announce that Xata now offers direct SQL access to
+your database.
+
+Internally, Xata relies on [PostgreSQL](/docs/concepts/how-it-works)
+for storing your data. We want you to experience the benefits of SQL
+beyond the existing Xata API. From now on we support running some SQL
+commands over HTTP. The allowed statements are `SELECT`, `INSERT`,
+`UPDATE` and `DELETE` with some limitations.
+
+So if there is something that is not covered by the Xata API, you also
+have the flexibility to execute your queries using the conventional
+Xata approach - using either the SDK or our REST API. Each request can
+contain a single SQL statement; currently, support for multiple
+statements is not available. You can submit parameterized queries with
+parameters, so you are protected from SQL injection attacks.
+
+<TabbedCode tabs={['TypeScript', 'Python', 'JSON']}>
+
+```ts
+const { records: myRecord } = await xata.sql<TeamsRecord>`SELECT * FROM teams`;
```

Comment:
```suggestion
const { records: myRecord } = await xata.sql<TeamsRecord>`SELECT * FROM "teams"`;
```

Reply:
Is that strictly required? If the table is lowercase "teams", that should work still, right?

Reply:
Ah, we do quote in the python and JSON, so for consistency it does make sense.

Reply:
Yeah I added it for consistency